src/main/java/io/javalin/plugin/openapi/dsl/openApiBuilderDsl.kt

```diff
@@ -92,7 +92,16 @@ fun Operation.applyMetaInfo(options: CreateSchemaOptions, path: PathParser, meta
         }
     }
 
-    if (documentation.hasResponses()) {
+    if (!documentation.hasResponses()) {
+        updateResponses {
+            updateResponse("200") {
+                this.description(
+                    "This endpoint currently has no documented responses, since OpenApi requires a response" +
+                            " which should be successful, this documentation was automatically generated"
```

Comment:
I don't think we should add this description. I think if another plugin (outside the control of the user) defines a path, this you be really annoying.
I would prefer a simple "Default response". What do you think?

Reply:
This is a good idea, but I think we should add a (one time) log message containing a warning that endpoints have undocumented responses. What do you think?

Reply:
@maxemann96 Sounds good to me!